pkg/kubectl/describe.go

```diff
@@ -775,6 +786,27 @@ func describeContainers(pod *api.Pod, out io.Writer) {
 	}
 }
 
+func describeProbe(probe *api.Probe) string {
```

Comment:
Happy to export it.

On Wed, Feb 17, 2016 at 5:37 AM, Michail Kargakis <notifications@github.com>
wrote:

> In pkg/kubectl/describe.go
> <https://github.com/kubernetes/kubernetes/pull/21341#discussion_r53146418>
> :
>
> > @@ -775,6 +786,27 @@ func describeContainers(pod *api.Pod, out io.Writer) {
> >  	}
> >  }
> >
> > +func describeProbe(probe *api.Probe) string {
>
> If you are not going to refactor describeContainers by 1) making it work
> w/o accepting a pod, 2) export it, then we should at least export this
> helper.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/21341/files#r53146418>.
>